apps/iframe/src/Components/Modal/Modal.tsx

```diff
@@ -0,0 +1,84 @@
+import React, { useState, useEffect } from 'react'
+import './Modal.scss'
+
+import Header from '../Header/Header'
+import Content from '../Content/Content'
+
+// const didShorten = (did: string): string => `${did.substring(0, 10)}...${did.substring(did.length - 5, did.length)}`
+
+type ModalProps = {
+  type?: string
+}
+
+// TODO: Implement Error component
+
+export const Modal = ({ type }: ModalProps) => {
+  const [modal, setModal] = useState(
```

Comment:
Why is this set in state please?

Reply:
I set the wrong body here, wanted to set our "detecting request..." instead. Will update.

Reply:
I mean why are there elements put in state?